### DISCUSSION TITLE
Can I get shell access to a machine with GPUs with Akash network?

### DISCUSSION THREAD

**Discussion by mashdragon** [akash-network/community#665] [Q&A] [answered]
I would like to do some AI prototyping, and Akash network has a decent GPU selection. I am wondering if there is a simple existing way to get SSH access to a shell on a Akash-provided machine having access to GPUs so I can do my development from there for some amount of time, maybe a few hours, and only pay for those few hours. I found when briefly going through the Console application that you can deploy Docker containers on Akash. I would rather just have shell access to a Linux OS, but maybe there is a Docker image or some other way to do this that has already been developed?

**Comment by dominikusbrian** [comment, marked as answer]
Hi @mashdragon feel free to try the official new Plain Linux option from console.akash.network 
Go to console.akash.network, click the big red button on the left sidebar saying "Deploy" and you will got to this page shown below
![image](https://github.com/user-attachments/assets/80ac89cf-9502-45f7-b8d9-b93daca4a8b8)
choose the plain linux and you can work with your SSH setup from there, generate key-value pair, and pre-populating the instance, etc. 

Feel free to continue this discussion on the Deployment Channels of Akash Discord server https://discord.com/channels/747885925232672829/1111748832489910332/1271461753376477265 
Many more examples from the community there.

**Reply by mashdragon** [reply]
Excellent, thank you. Discord is currently blocking me from joining new servers for some reason, so I cannot view that.

**Reply by ghost** [reply]
Hey Mash! Are you still having issues joining Akash discord?

**Reply by mashdragon** [reply]
Hi there! Yes, this is a Discord issue I'm afraid. I have been unable to join any new servers and get a vague error message after it prompts me to complete a captcha. It surprises me that so many blockchain communities rely on such a highly centralized, walled off platform for discussion. Discord support does not reply, and I have just about given up trying.

**Reply by pcfreak30** [reply]
> Hi there! Yes, this is a Discord issue I'm afraid. I have been unable to join any new servers and get a vague error message after it prompts me to complete a captcha. It surprises me that so many blockchain communities rely on such a highly centralized, walled off platform for discussion. Discord support does not reply, and I have just about given up trying.

Its a bit random of a comment, but I can also agree with this and have thought the same.
